app/assets/stylesheets/player.scss

```diff
@@ -0,0 +1,60 @@
+.container-sign-up{
+  .div-btn {
+    text-align: center;
+  }
+
+  .btn-style {
+    width: 100%;
+    background-color: $secondary-font-color;
+    &:hover {
+      background-color: $secondary-font-color-hover;
+    }
+  }
+
+  .link-forgot-password {
+    color: $white;
+  }
+  
+  .link-sign-in {
+    margin-top: 15px;
+  }
+  
+  .col-sign-up {
+    width: 450px;
+    @media(max-width: 768px) {
+      margin: 0;
+     }
+  }
+
+  .col-forgot-pass {
+    width: 450px;
+    @media(max-width: 768px) {
```

Comment:
Can you add a `mixin` here. You can create a file in `abstracts/_mixins.scss`.  Then import this file/folder in you `aplication.scss` (Make sure this is imported before all the rest of the files, so it can be used by any other stylesheet).

Replace this line with something like `@include breakpoint($breakpointName) { Styles }`

For more information about breakpoints, here is a good article: https://responsivedesign.is/develop/getting-started-with-sass-and-breakpoint-mixin/